IntroToCSharpSite.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": "."
+		},
+		{
+			"path": "IntroToCSharp/wwwroot/examples/csharp"
+		}
+	],
+	"settings": {}
+}
```

Comment:
This file should not be added. You should either delete it locally or add it to .gitignore.